mapdata/WarcraftLegacies/UpgradeData/Skin/927019090.json

```diff
@@ -0,0 +1,45 @@
+{
+  "OldId": 1701079122,
+  "NewId": 927019090,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Level": 1,
+      "Id": 829579623,
+      "Type": 3,
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNMassTeleport.blp"
```

Comment:
Do we want Mass Teleport as the icon? It seems misleading to me; makes it look like a research that gives you mass Teleport.

Reply:
It was in the past when this exact mechanic existed. But if you think its better, il add a new one!

Reply:
Ahh that makes sense. Yeah I think it's going to be way clearer with a special icon.